src/components/TheDetails.vue

```diff
@@ -587,6 +589,13 @@ export default {
 				return 'icon-bw icon-task-star'
 			}
 		},
+		iconPinned: function() {
+			if (this.task.pinned) {
+				return 'icon-bw icon-pinned-off'
```

Comment:
```suggestion
				return 'icon-bw icon-pinned'
```

Reply:
I think it is a bit confusing that the opposite icon is shown in the Task body and the details header. And we normally use the icon indicating the current state, and not the state you will achieve clicking the icon (e.g. the starred icon is red for important tasks).

Reply:
Sorry for that, i copied the behaviour of OpenTasks at that point :)